packages/radix-ui-themes/src/components/base-tab-list.css

```diff
@@ -0,0 +1,140 @@
+.rt-BaseTabList {
+  display: flex;
+  overflow-x: auto;
+  white-space: nowrap;
+
+  scrollbar-width: none;
+  &::-webkit-scrollbar {
+    display: none;
+  }
+}
+
+.rt-BaseTabListLink {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  flex-shrink: 0;
+  position: relative;
+  user-select: none;
+}
+
+.rt-BaseTabListLinkInner,
+.rt-BaseTabListLinkInnerHidden {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.rt-BaseTabListLinkInner {
+  position: absolute;
+
+  :where(.rt-BaseTabListLink[data-state='inactive'], :not(.rt-TabNavLink[data-active])) & {
+    letter-spacing: var(--tabs-trigger-inactive-letter-spacing);
```

Comment:
Not sure what to do about all the `--tabs-trigger-*` variables?

Reply:
Let's keep them as is

Reply:
I think we are fine given that our alternatives are to either have two sets of variables for components with matching design, or to rename this set of variables into something that would work for both but then deal with this being a breaking change.

It's a bit unfortunate we haven't foreseen that we'd add another tabs component, but those tab nav links are also triggers in a way. I can take a note to return to this area again when we do a release with breaking changes. Unless you had something else in mind?